index.html

```diff
@@ -8,32 +8,171 @@
-        <img class="network-battery" src="img\Group 3.svg" alt="network and battery">
-    </section>
-    <header class="header">
-        <a href="#"><img class="logo" src="img\Logo Placeholder.svg"></a>
-        <nav class="menu">
-            <img class="bar" src="img\Icon - Menu.svg" alt="hambuger bar menu">
-        </nav>
-    </header>
-    <section class="content">
-        <h1 class="headline">I’m Mariane Glad to see you!</h1>
-        <p class="description">
-            I’m a software developer! I can help you build a product , 
-            feature or website Look through some of my work and experience! 
-            If you like what you see and have a project you need coded, 
-            don’t hestiate to contact me.
-        </p>
-        <h2 class="social">LET’S CONNECT</h2>
-        <ul class="social-icon">
-            <li><a href="#"><img class="icon" src="img\Vector.svg" alt="Tweeter icon"></a></li>
-            <li><a href="#"><img class="icon" src="img\Linkedin icon.svg" alt="Linkedin icon"></a></li>
-            <li><a href="#"><img class="icon" src="img\Vector m.svg" alt="M icon"></a></li>
-            <li><a href="#"><img class="icon" src="img\github icon.svg" alt="github icon"></a></li>
-            <li><a href="#"><img class="icon" src="img\peace icon.svg" alt="peace icon"></a></li>
-        </ul>
-    </section>
+    <div id="presentation">
+        <section class="status-bar">
+            <img class="time" src="img\Time Style.svg" alt="time">
+            <img class="network-battery" src="img\Group 3.svg" alt="network and battery">
+        </section>
+        <header class="header">
+            <a href="#"><img class="logo" src="img\Logo Placeholder.svg"></a>
+            <nav class="menu">
+                <img class="bar" src="img\Icon - Menu.svg" alt="hambuger bar menu">
+            </nav>
+        </header>
+        <section class="content">
+            <h1 class="headline">I’m Mariane Glad to see you!</h1>
+            <p class="description">
+                I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+            <h2 class="social">LET’S CONNECT</h2>
+            <ul class="social-icon">
+                <li><a href="#"><img class="icon" src="img\Vector.svg" alt="Tweeter icon"></a></li>
+                <li><a href="#"><img class="icon" src="img\Linkedin icon.svg" alt="Linkedin icon"></a></li>
+                <li><a href="#"><img class="icon" src="img\Vector m.svg" alt="M icon"></a></li>
+                <li><a href="#"><img class="icon" src="img\github icon.svg" alt="github icon"></a></li>
+                <li><a href="#"><img class="icon" src="img\peace icon.svg" alt="peace icon"></a></li>
+            </ul>
+        </section>
+    </div>
+    <div id="work">
```

Comment:
- **[OPTIONAL]** Consider changing the main container of your **Work section** into `<section>`.